src/diffpy/utils/parsers/loaddata.py

```diff
@@ -98,6 +134,11 @@ def countcolumnsvalues(line):
             # in case of trailing delimiters.
             kwargs.setdefault('usecols', list(range(ncvblock[0])))
             rv = loadtxt(fid, **kwargs)
+
+    # return headers if requested
+    if headers:
+        return hdata, rv
+    # otherwise do not
```

Comment:
do we need an else: here?

Reply:
no need for else statement since the return logic takes care of it

Reply:
OK.  It may still be good to have it just so the intent is clear to future developers like me who can't quite remember the default return logic... ;)  I guess it returns None in that case, right?

Reply:
my guiding philosophy on htings like that is to always do things to save human brain time, at the expense of extra work for the computer, unless there is a significant performance hit.....   As you do more programming you realize the human brain time is way more valuable and limited resource....